### ISSUE TITLE
RxFormArray setValue

### ISSUE THREAD

**Issue by VoteForPedro** [rxweb/rxweb#524]
Is there a way to replace existing formarray controls? 

I have RxFormArray that is empty, When im receiving data, im trying to set it to form array:
`
var eqs = (this.form.get("equipments") as RxFormArray);
let items = result.equipments.map((a) => this.formBuilder.formGroup(new EquipmentViewModel(a)));
eqs.setValue(items);
`
setValue function call throws error:
_There are no form controls registered with this array yet. If you're using ngModel,
        you may want to check next tick (e.g. use setTimeout)._

@rxweb/reactive-form-validators: "2.1.4"

**Comment by ajayojha** [comment]
@VoteForPedro How you are creating the nested FormGroup? please share the little bit code snippet.

**Comment by VoteForPedro** [comment]
`
export class EquipmentViewModel {
  @unique()
  id: string;

  @required()
  equipmentName: string;

  constructor(summary?: OrderEquipmentSummary) {
    this.id = summary?.id;
    this.equipmentName = summary?.equipmentName;
  }
}

export class OrderViewModel {
  @propArray(EquipmentViewModel)
  equipments: EquipmentViewModel[];

  constructor(detail?: OrderDetail) {
    this.equipments = detail?.equipments.map((eq) => {
      return new EquipmentViewModel(eq);
    }) ?? [];
  }
}

...

let viewModel = new OrderViewModel();
this.form = this.formBuilder.formGroup(viewModel) as RxFormGroup;
`

**Comment by ajayojha** [comment]
@VoteForPedro uses the push method of 'RxFormArray'.

here is the example:
```js
var eqs = (this.form.get("equipments") as RxFormArray);
result.equipments.forEach((a) => eqs.push(this.formBuilder.formGroup(new EquipmentViewModel(a))));
```

**Comment by VoteForPedro** [comment]
@ajayojha yes, im already using push. But how can i replace an array of existing controls?
